pwcnet/context_network/model.py

```diff
@@ -29,7 +30,7 @@ def __init__(self, name='context_network', layer_specs=None,
         else:
             self.layer_specs = layer_specs
 
-    def get_forward(self, features, optical_flow, reuse_variables=False):
+    def get_forward(self, features, optical_flow, reuse_variables=tf.AUTO_REUSE):
```

Comment:
Should reuse_variables have tf.AUTO_REUSE as a default from now on? I think that False as a default is safer for me, since on multiple occasions I have accidentally used the same scopes (when writing tests).

Reply:
Should be safe in these cases.